core/google/cloud/_helpers.py

```diff
@@ -34,7 +34,7 @@
     import grpc
     from google.auth.transport.grpc import (
         AuthMetadataPlugin)  # pragma: NO COVER
-except ImportError:
+except ImportError:  # pragma: NO COVER
```

Comment:
But the presence of your `# pragma: NO COVER` here indicates `grpc` **is** present in the relevant environment